CONTRIBUTING.md

```diff
@@ -42,6 +43,17 @@ To submit a bug, [create new issue](https://jira.hyperledger.org/secure/CreateIs
 | Environment             | Your OS, device's specs, Virtual Environment if you use one, version of Iroha etc. |
 
 
+### Reporting Vulnerabilities
+
+While we try to be proactive in preventing security problems, we do not assume they’ll never come up.
```

Comment:
Double negative seems confusing. Maybe put it like "Although we try to be proactive in preventing security problems, we cannot ignore the possibility" or smth like that

Reply:
TBH, I reused text from Wordpress website here: https://make.wordpress.org/core/handbook/testing/reporting-security-vulnerabilities/

Reply:
Ok ) I've just got confused myself when I read it.